proposals/lip-0028.md

```diff
@@ -31,37 +31,37 @@ When a transaction object is serialized, also the asset property has to be seria
 
 ![Transaction Serialization Diagram](lip-0028/transaction_serialization_diagram.png)
 
-For this reason, this LIP proposes to divide the serialization-deserialization process. First it defines a serialization-deserialization process for the whole transaction object with the `baseTransaction` JSON schema where the `asset` property is assumed to be of type `bytes`. Secondly, it defines the serialization-deserialization process for the `asset` property with the corresponding `asset` property JSON schema depending on the type of the transaction (specified in the `type` property). There are 7 different `asset` property schemas, one for each of the transaction types in the Lisk protocol. In order to simplify the creation of custom transactions with the SDK and to make the validation process less error prone, every property defined inside a given `asset` schema is required.
+For this reason, this LIP proposes to divide the serialization-deserialization process. First it defines a serialization-deserialization process for the whole transaction object with the `transactionSchema` JSON schema where the `asset` property is assumed to be of type `bytes`. Secondly, it defines the serialization-deserialization process for the `asset` property with the corresponding `asset` property JSON schema depending on the type of the transaction (specified by the unique combination of `moduleID` and `assetID` properties). There are 7 different `asset` property schemas, one for each of the transaction types in the Lisk protocol. In order to simplify the creation of custom transactions with the SDK and to make the validation process less error prone, every property defined inside a given `asset` schema is required.
 
 ## Specification
 
-Assuming that the JSON schemas for the `baseTransaction` and for the `asset` properties are provided according to the specifications in the [LIP 0027](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0027.md), the serialization and deserialization of a transaction object are done as follows.
+Assuming that the JSON schemas for the `transactionSchema` and for the `asset` properties are provided according to the specifications in the [LIP 0027](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0027.md), the serialization and deserialization of a transaction object are done as follows.
 
 ### Serialization
 
 Consider a data structure `trsData` representing a valid (signed or unsigned) transaction object to be serialized. The serialization procedure is done in 3 steps:
 
-1. The `trsData.asset` property is serialized to bytes according to the corresponding  `asset` property schema specified by the `trsData.type` property (see [below](#baseTransaction-schema)).
+1. The `trsData.asset` property is serialized to bytes according to the corresponding  `asset` property schema specified by the combination of `trsData.moduleID` and `trs.assetID` properties (see [below](#transactionSchema-schema)).
```

Comment:
In a few places, you use `trsData.moduleID` and `trs.assetID`. Why is it not `trsData.assetID` ?

Reply:
when you say combination, you mean literally the concatenation of the values? Because in principle this sounds as we could do any operation taking these 2 values as input. Could we maybe explicitly state how this combination is taken?

Reply:
those are mistakes @MaximeGagnebin  bad copy paste habits.

Reply:
This is specified in the new  LIP 0033. I added the link to the specs of LIP 0033 that hopefully should be updated not much later we update this one (assuming it is not a new LIP). @ricott1